### DISCUSSION TITLE
How to Position Hover Div Relative to a Movable Node in React Flow?

### DISCUSSION THREAD

**Discussion by shubham-smartfoodsafe** [xyflow/xyflow#4843] [Q&A]
I'm working with React Flow and I'm using the **onNodeMouseEnter** and **onNodeMouseLeave** events to show a div with additional information when a particular node is hovered. 
The problem I’m encountering is that when I hover over a node, the div appears, but it doesn't position relative to the hovered node. Since the node is movable, the hover div stays in a static position and doesn't move with the node.

I’ve tried implementing this, and here’s a **CodeSandbox** :  https://codesandbox.io/p/live/7d64285b-ab06-4ca0-ad57-3e4999cfc691

How can I ensure the div remains positioned relative to the node, even as the node moves around?

https://github.com/user-attachments/assets/09024646-4eda-4d98-b2d5-692096492a1d

**Comment by vinhphanybn1** [comment]
I can't access your CodeSandBox, but you can make it easily with custom Node. In my case, I'm using Tailwind, and it works normally:
```
 <div onDoubleClick={handleDoubleClick}
          className="group transition duration-150 ease-in-out transform">
          <span>{label}</span>
          <div className="hidden group-hover:flex -top-5 -translate-y-1/2 -right-[5px] absolute items-center">
            <RiUserFill className="h-4 w-4"></RiUserFill> Hover content
          </div>
        </div>
        ```
        
I hope this will help you
